src/actions/subject.js

```diff
@@ -0,0 +1,110 @@
+import apiClient from 'panoptes-client/lib/api-client'
+import { isNil, length, prepend } from 'ramda'
+import { setState } from '../actions/index'
+import { Image } from 'react-native'
+import getSubjectLocation from '../utils/get-subject-location'
+
+export function fetchUpcomingSubjects(workflowID) {
+  return dispatch => {
+    return new Promise ((resolve, reject) => {
+      apiClient.type('subjects').get({workflow_id: workflowID, sort: 'queued'}).then((subjects) => {
```

Comment:
`sort: "queued"` results in the same request as using `/api/subjects/queued`.  

See https://github.com/zooniverse/Panoptes/blob/master/app/controllers/api/v1/subjects_controller.rb#L13